yt_dlp/extractor/afreecatv.py

```diff
@@ -311,6 +312,16 @@ def _real_extract(self, url):
                 key = file_element.get('key', '')
                 upload_date = self._search_regex(
                     r'^(\d{8})_', key, 'upload date', default=None)
+                if upload_date is not None:
```

Comment:
The parsed upload date should simply be passed through `unified_strdate` which would avoid this issue. You can still keep the sanity check on year if needed

Reply:
Ah, didn't see `unified_strdate` - that's definitely the right call here, thanks!

If we do keep the sanity check, is there another function I should be using to parse the resulting string, or is `date_from_str` still okay?

Reply:
yeah, `date_from_str` is fine. But no need to catch any exception